source/renderer/app/components/staking/stake-pools/StakePoolsRanking.js

```diff
@@ -0,0 +1,316 @@
+import { shortNumber } from '../../../utils/formatters';
+import Wallet from '../../../domains/Wallet';
+import ButtonLink from '../../widgets/ButtonLink';
+import Slider from '../../widgets/Slider';
+import styles from './StakePoolsRanking.scss';
+
+const messages = defineMessages({
+  rankingAllWallets: {
+    id: 'staking.stakePools.rankingAllWallets',
+    defaultMessage: '!!!all your wallets',
+    description: 'All wallets item of dropdown.',
+  },
+  rankingAllWalletsEnd: {
+    id: 'staking.stakePools.rankingAllWalletsEnd',
+    defaultMessage: '!!!.',
+    description: 'All wallets description after dropdown.',
+  },
+  rankingAllWalletsStart: {
+    id: 'staking.stakePools.rankingAllWalletsStart',
+    defaultMessage:
+      '!!!Stake pools are currently ranked based on the combined amount in',
+    description: 'All wallets description before dropdown.',
+  },
+  rankingDescription: {
+    id: 'staking.stakePools.rankingDescription',
+    defaultMessage:
+      '!!!Use the slider to rank the stake pools based on the amount you intend to delegate.',
+    description: 'Ranking description.',
+  },
+  rankingOneWalletEnd: {
+    id: 'staking.stakePools.rankingOneWalletEnd',
+    defaultMessage: '!!!wallet.',
+    description: 'One wallet description after dropdown.',
+  },
+  rankingOneWalletStart: {
+    id: 'staking.stakePools.rankingOneWalletStart',
+    defaultMessage:
+      '!!!Stake pools are currently ranked based on the amount in',
+    description: 'One wallet description before dropdown.',
+  },
+  rankingSelectWallet: {
+    id: 'staking.stakePools.rankingSelectWallet',
+    defaultMessage: '!!!select a wallet',
+    description: 'Select wallet item of dropdown.',
+  },
+  rankingSelectWalletEnd: {
+    id: 'staking.stakePools.rankingSelectWalletEnd',
+    defaultMessage: '!!!to set the amount you intend to delegate.',
+    description: 'Select wallet description after dropdown.',
+  },
+  rankingSelectWalletStart: {
+    id: 'staking.stakePools.rankingSelectWalletStart',
+    defaultMessage: '!!!Or',
+    description: 'Select wallet description before dropdown.',
+  },
+  rankingExtraTooltip: {
+    id: 'staking.stakePools.rankingExtraTooltip',
+    defaultMessage: '!!!Circulating supply',
+    description: 'Circulating supply slider tooltip.',
+  },
+  rankingMaxTooltip: {
+    id: 'staking.stakePools.rankingMaxTooltip',
+    defaultMessage: '!!!Saturation point',
+    description: 'Saturation point slider tooltip.',
+  },
+  rankingMinTooltip: {
+    id: 'staking.stakePools.rankingMinTooltip',
+    defaultMessage: '!!!Minimum ADA required for staking',
+    description: 'Minimum ADA required for staking slider tooltip.',
+  },
+  actionLearnMore: {
+    id: 'staking.stakePools.learnMore',
+    defaultMessage: '!!!Learn more',
+    description: 'Learn more action of ranking panel.',
+  },
+});
+
+const learnMoreUrlMap = {
+  'en-US': 'https://iohk.zendesk.com/hc/en-us',
+  'ja-JP': 'https://iohk.zendesk.com/hc/ja',
+};
```

Comment:
@yakovkaravelov this should be included inside translations and pulled with `intl`

Reply:
👍